pom.xml

```diff
@@ -241,6 +241,32 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
```

Comment:
In main pom.xml , already exists rat plugin , can you just update it:
<plugin>
            <groupId>org.apache.rat</groupId>
            <artifactId>apache-rat-plugin</artifactId>
            <version>0.11</version>
            <executions>
              <execution>
                <phase>verify</phase>
                <goals>
                  <goal>check</goal>
                </goals>
              </execution>
            </executions>
            <configuration>
              <reportFile>${project.build.directory}/${project.build.finalName}.rat</reportFile>
              <excludeSubProjects>false</excludeSubProjects>
              <useDefaultExcludes>true</useDefaultExcludes>
              <excludes>
                <exclude>**/*.iml</exclude>
                <exclude>**/*.csv</exclude>
                <exclude>**/*.dictionary</exclude>
                <exclude>**/*.ktr</exclude>
                <exclude>**/*.rat</exclude>
                <exclude>**/_SUCCESS</exclude>
                <exclude>**/non-csv</exclude>
                <exclude>**/.invisibilityfile</exclude>
                <exclude>**/noneCsvFormat.cs</exclude>
                <exclude>**/org.apache.spark.sql.sources.DataSourceRegister</exclude>
                <exclude>**/org.apache.spark.sql.test.TestQueryExecutorRegister</exclude>
                <exclude>**/derby.log</exclude>
                <exclude>**/meta.lock</exclude>
                <exclude>**/loadmetadata.metadata</exclude>
                <exclude>**/modifiedTime.mdt</exclude>
                <exclude>**/PULL_REQUEST_TEMPLATE.md</exclude>
                <exclude>**/dict.txt</exclude>
                <exclude>**/sample</exclude>
                <exclude>**/test.json</exclude>
              </excludes>
            </configuration>
          </plugin>